INSTALL.md

```diff
@@ -0,0 +1,50 @@
+# Metrics deployment instructions
+## Create app
+```bash
+dokku$ dokku apps:create metrics
+dokku$ dokku git:set metrics deploy-branch main
+```
+
+## Configure app
+```bash
+dokku config:set metrics GITHUB_TOKEN'xxx'
+dokku config:set metrics SLACK_SIGNING_SECRET='xxx'
+dokku config:set metrics SLACK_TECH_SUPPORT_CHANNEL_ID='xxx'
+dokku config:set metrics SLACK_TOKEN='xxx'
+dokku config:set metrics TIMESCALEDB_URL='xxx'
+```
+
+## Disable checks
+Dokku performs health checks on apps during deploy by sending requests to port 80.
+This tool isn't a web app does it can't accept requests on a port.
+Disable the checks so deploys can happen.
+```bash
+dokku$ dokku checks:disable metrics
+```
+
+## Set up storage
```

Comment:
weird that GH hasn't syntax highlighted this correctly - does it need newline after the title line to parse correctly as GitHubMarkdown? 
```suggestion
## Set up storage

```

Reply:
Arg.  It's whatever not-actually-a-space character that option+space creates on macOS.